hasura/graphql-engine
# console: track all tables and relationships optimisation (close #5014)

### Description
When Hasura is connected to the existing PG instance, tracking all tables were taking a while to complete the api request(with bulk queries/each table queries). This PR will simplify this by manually creating metadata from the UI and replace on the server (only if the metadata is empty in the server)

### Changelog


### Affected components


### Related Issues
closes  #5014 

### Solution and Design
New Approach
1. The user clicks the `Track All` button. (Tables or Relationships)
2. Check if the server metadata is empty or not (`export_metadata`)
3. if server metadata is empty, generate metadata from the front end app and replace metadata directly onto the server.
4. if there is any failure, fallback to existing logic. 

### Steps to test and verify
1. Make sure Hasura metadata is empty and have some untracked tables
> in case of testing relationships, make sure that metadata contains only table information.
2. Click on the `Track All` tables button. (Tables/Relationships)
3. compare & validate the API call with the previous version. now it should call `replace_metadata` instead of multiple `track_tables`
